### PR TITLE
Updated gettext functions

### PR DESCRIPTION
**Pull request by dnaluz**
A lot of code was updated here.  A few strings were missed initially and some code had to be refactored.  Please make sure that all of the event functionality still works across the site and you are still able to edit and create events with all labels showing properly and data showing properly across the entire site in all sections.

### CODE REVIEW

File: templates/share-modal.php [hunk 1]
```diff
@@ -9,31 +9,31 @@
         </svg>
     </button>
     <div class="share-lightbox">
-        <p class="title--secondary"><?php echo __('Share', "community-portal") ?></p> 
+        <p class="title--secondary"><?php echo __('Share', 'community-portal') ?></p>
```
Comment (kycapstick) [inline templates/share-modal.php:12]: __() should be changed to _e()

File: templates/share-modal.php [hunk 2]
```diff
@@ -9,31 +9,31 @@
...
             <li class="share-link">
                 <a href="<?php echo esc_attr('https://discourse.mozilla.org/new-topic?title='.$url)?>" class="btn btn--light btn--share share-link__discourse">
-                    <?php echo __('Share to Discourse', "community-portal") ?>
+                    <?php _e('Share to Discourse', 'community-portal') ?>
                 </a>
             </li>
             <li class="share-link">
                 <a href="<?php echo esc_attr('https://telegram.me/share/url?url='.$url) ?>" class="btn btn--light btn--share share-link__telegram" >
-                    <?php echo __('Share to Telegram', "community-portal") ?>
+                    <?php echo __('Share to Telegram', 'community-portal') ?>
```
Comment (kycapstick) [inline templates/share-modal.php:36]: __() should be changed to _e()

File: templates/share-modal.php (hunk 1 above)
Comment (dnaluz) [reply templates/share-modal.php]: Updated

File: templates/share-modal.php (hunk 2 above)
Comment (dnaluz) [reply templates/share-modal.php]: Updated

**Pull request merged**